### PR TITLE
Bump versions for release 0.12.0.

### PR DESCRIPTION
**Pull request by dhatch**
PR checklist:

- [ ] Added changelog entry.

### CODE REVIEW

File: docs/themes/oso-tailwind/assets/css/styles.css [hunk 1]
```diff
@@ -96,3 +96,16 @@ button > * {
   height: 1rem;
   vertical-align: middle;
 }
+
+.prose h5 + * {
```
Comment (samscott89) [inline docs/themes/oso-tailwind/assets/css/styles.css:100]: In theory, these should be above in the components layer, and use `@apply`.

e.g.

```
.prose h5 + * {
   @apply mt-0;
}
```

but I'm not sure how to reverse engineer the other styles from [here](https://github.com/tailwindlabs/tailwindcss-typography/blob/95bb9eb2807465432dad98d083d7573c4c5b5962/src/styles.js#L395-L398)

File: docs/content/any/project/changelogs/TEMPLATE.md [hunk 2]
```diff
@@ -8,11 +8,11 @@ description: >-
 draft: true
 ---
 
-## `RELEASED_PACKAGE_1` NEW_VERSION
+# `RELEASED_PACKAGE_1` NEW_VERSION
```
Comment (gj) [inline docs/content/any/project/changelogs/TEMPLATE.md:11]: I had this started at h2 b/c you're only supposed to have a single h1 per page for accessibility & SEO purposes

**Review by samscott89 — APPROVED** [review]
LGTM!

File: docs/content/any/project/changelogs/TEMPLATE.md (hunk 2 above)
Comment (dhatch) [reply docs/content/any/project/changelogs/TEMPLATE.md:11]: o

Comment (dhatch) [reply docs/content/any/project/changelogs/TEMPLATE.md:11]: ok smarter than me

**Pull request merged**